deploy/registration-service.yaml

```diff
@@ -183,9 +240,29 @@ parameters:
     value: '3'
   - name: ENVIRONMENT
     value: 'prod' # prod, stage, e2e-tests, dev, etc
+  - name: CONFIG_MAP_NAME
+    value: ''
```

Comment:
As I have written above - there is no point in configuring the CM name

Reply:
see other comment